opossum-tool/OpossumUI
# chore: upgrade to Node.js 20

### Summary of changes

Upgrade to Node.js 20.

### How can the changes be tested

Check that the app starts correctly and pipeline is green.
You may also want to check that the VSCode devcontainer extension starts the repo in a container correctly. To do so, you need to install the extension and run "Dev Containers: Reopen in Container" from the VSCode command pallette.
